### PR TITLE
added lazy loaded module 'about'

### PR DESCRIPTION
**Pull request by d3ns0n**

### CODE REVIEW

File: src/app/about/about.component.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-about',
+  templateUrl: './about.component.html',
+  styleUrls: ['./about.component.css']
+})
+export class AboutComponent implements OnInit {
+
+  constructor() { }
```
Comment (wassertim) [inline src/app/about/about.component.ts]: you don't this :)

File: src/app/about/about.component.ts [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+  ngOnInit() {
```
Comment (wassertim) [inline src/app/about/about.component.ts]: you don't need that :)

**Comment by d3ns0n** [comment]
I realized I had the commit from the other branch in this as well. I removed them and fixed your remarks. Please check again.

File: src/app/about/about.component.html [hunk 3]
```diff
@@ -0,0 +1,19 @@
+<h1>Who are we?</h1>
```
Comment (wassertim) [inline src/app/about/about.component.html]: I think it is usually `who we are` without a question mark. Like here:
https://www.mckinsey.com/about-us/who-we-are

Comment (wassertim) [reply src/app/about/about.component.html]: I like the page. Maybe we should add some bootstrap styles? :)

File: src/app/about/about.component.html [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+      <div>
+        <img src="assets/wassertim.jpeg">
+        <br> Tim Tkachenko
+        <a href="https://github.com/wassertim/">wassertim@GitHub</a>
+      </div>
+    </td>
+    <td>
+      <div>
+        <img src="assets/d3ns0n.jpeg">
+        <br> Dennis Lotz
```
Comment (wassertim) [inline src/app/about/about.component.html]: don't use br just wrap it inside div :)

File: src/app/about/about.component.html [hunk 5]
```diff
@@ -0,0 +1,19 @@
+<h1>Who are we?</h1>
+<table>
+  <tr>
+    <td>
+      <div>
+        <img src="assets/wassertim.jpeg">
+        <br> Tim Tkachenko
```
Comment (wassertim) [inline src/app/about/about.component.html]: same here

**Comment by wassertim** [comment]
Looks great! Let's merge it

**Comment by wassertim** [comment]
I fink you no mind me merge it :)

**Pull request merged**